src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -201,13 +201,13 @@ public net.minecraft.server.level.ServerPlayer m_143399_(Lnet/minecraft/world/in
 public net.minecraft.server.level.ServerPlayer m_9217_()V # nextContainerCounter
 public net.minecraft.server.packs.AbstractPackResources f_10203_ # file
 public net.minecraft.server.packs.resources.FallbackResourceManager f_10599_ # fallbacks
-public net.minecraft.tags.Tag$BuilderEntry <init>(Lnet/minecraft/tags/Tag$Entry;Ljava/lang/String;)V # constructor
 public net.minecraft.tags.BlockTags m_13116_(Ljava/lang/String;)Lnet/minecraft/tags/Tag$Named; # bind
 public net.minecraft.tags.EntityTypeTags m_13127_(Ljava/lang/String;)Lnet/minecraft/tags/Tag$Named; # bind
 public net.minecraft.tags.FluidTags m_13134_(Ljava/lang/String;)Lnet/minecraft/tags/Tag$Named; # bind
 public net.minecraft.tags.GameEventTags m_144307_(Ljava/lang/String;)Lnet/minecraft/tags/Tag$Named; # bind
 public net.minecraft.tags.ItemTags m_13194_(Ljava/lang/String;)Lnet/minecraft/tags/Tag$Named; # bind
 protected net.minecraft.tags.StaticTagHelper$Wrapper f_13251_ # tag
+public net.minecraft.tags.Tag$BuilderEntry <init>(Lnet/minecraft/tags/Tag$Entry;Ljava/lang/String;)V # constructor
```

Comment:
I presume this is an automated change by `checkATs`?

Reply:
Yep.